instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/OpenTelemetryAppenderConfigTest.java

```diff
@@ -61,7 +61,6 @@ static void setupAll() {
             .addLogRecordProcessor(SimpleLogRecordProcessor.create(logRecordExporter))
             .build();
 
-    GlobalOpenTelemetry.resetForTest();
```

Comment:
Nit: can you revert that change? Even if tests pass without that, I think it's a good practice to reset the global state (e.g. in case more tests get added in the future)